packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -194,11 +194,14 @@ export default class TableResize implements EditorPlugin {
                                 }
                                 if (verticalInserterTd) {
                                     this.setCurrentTd(null);
-                                    this.setCurrentInsertTd(
-                                        ResizeState.Vertical,
-                                        verticalInserterTd,
-                                        map.rect
-                                    );
+                                    // we hide the inserter if left mouse button is pressed
+                                    if (e.buttons !== 1) {
```

Comment:
what if I press both left and right button? Will buttons be 2? I prefer check buttons == 0 here